packages/pwa-kit-dev/src/utils/resolver-utils.js

```diff
@@ -85,26 +89,32 @@ export const expand = (extensions = []) =>
  * @param {String} importPath - The import module-name.
  * @param {String} sourcePath - The path the the file of the source import.
  * @param {Object} opts - The path the the file of the source import.
- * @param {Array<shortName: String, config: Array>} opts.extensions - List of extensions used by the base PWA-Kit application.
+ * @param {Array<shortName: String, config: Array>} opts.extensionEntries - List of extension entries (tupals) used by the base PWA-Kit application.
  * @param {String>} opts.projectDir - Absolute path of the base project.
  * @returns {String[]} paths - The potential paths to find the module import.
  */
 export const buildCandidatePaths = (importPath, sourcePath, opts = {}) => {
     // Replace wildcard character as it has done its job getting us to this point.
     importPath = importPath.replace('*/', '')
 
-    const {extensions = [], projectDir = process.cwd()} = opts
+    const {extensionEntries = [], projectDir = process.cwd()} = opts
     const isSelfReferenceImport = isSelfReference(importPath, sourcePath)
-
-    let paths = expand(extensions).reverse()
+    let paths = []
 
     // Map all the extensions and resolve the module names to absolute paths.
-    paths = paths.reduce((acc, extensionEntry) => {
-        // The reference can be a module/package or an absolute path to a file.
-        const [extensionRef] = extensionEntry
-        const srcPath = path.join(projectDir, NODE_MODULES, extensionRef, SRC)
-        return [...acc, path.join(srcPath, OVERRIDES, importPath), path.join(srcPath, importPath)]
-    }, [])
+    paths = expand(extensionEntries)
```

Comment:
Should we validate the paths exsist before processing them?

Reply:
No. This already happen internally in the `resolve` call so we can just lean on it to do the right thing.